src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -138,5 +140,13 @@ public override void PlatformArrange(Rect frame)
 			this.PrepareForTextViewArrange(frame);
 			base.PlatformArrange(frame);
 		}
+
+		void OnFocusedChange(object? sender, FocusChangeEventArgs e)
+		{
+			if (VirtualView == null)
```

Comment:
is null over == null. == and != can be overridden but is cannot. If == is bad implemented you'll have a ver strange error, is removes that concern.

Reply:
@MartyIX yes, but if you add new code, it should be using the correct pattern/code. Old code will have to be tackle in the issue. But let's not add more work for us in the future.